### PR TITLE
Vxlan ecmp primary/secondary switchover tests.

### PR DESCRIPTION
**Pull request by siqbal1986**
### Description of PR

This PR adds the tests for priority tunnels where switchover from primary-secondary endpoint sets is performed upon failure of endpoint.
Summary:
Fixes # (issue)

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

### Back port request
- [ ] 201911
- [X] 202012
- [X] 202205

#### Any platform-specific information?
These tests only run on platforms that support Vxlan tunnels, such as Cisco 8102, Mlnx 4600, Mlnx 2700
#### Supported testbed topology if it's a new test case?
t1-64-lag

### CODE REVIEW

File: tests/vxlan/test_vxlan_ecmp.py [hunk 1]
```diff
@@ -1970,3 +1970,425 @@ def test_vxlan_varying_src_ip(self, setUp, encap_type):
...
+                                              tc1_end_point_list[1], "up")
+            time.sleep(10)
+            # verifying overlay_dmac
+            result = \
+                self.setup['duthost'].shell("sonic-db-cli APPL_DB HGET 'VNET_MONITOR_TABLE:{}:{}/{}' 'overlay_dmac'"
+                                            .format(
+                                                tc1_end_point_list[0],
+                                                tc1_new_dest,
+                                                ecmp_utils.HOST_MASK[ecmp_utils.get_payload_version(encap_type)]))
+            assert result['stdout'] == u'25:35:45:55:65:75'
```
Comment (prsunny) [inline tests/vxlan/test_vxlan_ecmp.py:2059]: can you define this as a variable instead of hardcode?

File: tests/vxlan/test_vxlan_ecmp.py [hunk 2]
```diff
@@ -1970,3 +1970,425 @@ def test_vxlan_varying_src_ip(self, setUp, encap_type):
...
+            7) bring B` down.
+            8) send packets to the route 1's prefix dst. packets are recieved at A`.
+            9) bring A up.
+            10) send packets to the route 1's prefix dst. packets are recieved at A.
+            11) bring B,C,C` up.
+            12) send packets to the route 1's prefix dst. packets are recieved at A,B.
+            13) Bring all endpoints down.
+            14) no traffic being passed.
+            15) bring B,C,C` up.
+            16) send packets to the route 1's prefix dst. packets are recieved at A,B.
```
Comment (prsunny) [inline tests/vxlan/test_vxlan_ecmp.py:2150]: line 15 is not bringing up A. Is that a typo?
Also please add one more test to bring standby first and then active after all are down.

Comment (siqbal1986) [reply tests/vxlan/test_vxlan_ecmp.py:2152]: this was a typo. i have fixed it . added the extra test.

File: tests/vxlan/test_vxlan_ecmp.py (hunk 1 above)
Comment (siqbal1986) [reply tests/vxlan/test_vxlan_ecmp.py]: done

**Pull request merged**